src/containers/Register/index.js

```diff
@@ -0,0 +1,152 @@
+import React, { Component } from 'react'
+
+import { white, indigo500, lightBlue400, lightBlue600, red600 } from 'material-ui/styles/colors'
```

Comment:
So we are going to use `material-ui` rather than implement more components in `toolset`?

Reply:
We are going to add more components for `toolset`, but separately from this application. `toolset` is not only for this app, but also for public usage. To boost up the speed of this app, we can use 3rd party library for UI components for the components we don't have. Again, `react-toolset` is not just for this application.